proto/ignite/services/plugin/grpc/v1/service.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package ignite.services.plugin.grpc.v1;
```

Comment:
Nit let's maybe change the namespace to apps already. So the protos aren't generated in a "plugin" folder

Reply:
I totally agree on that, thanks for pointing it out!

We are going finish the refactor from plugins to apps once this PR, #3544 and #3662 are merged. Having these issues merged will allow us to proceed with the final refactor step which is to change the source code from "Plugin" to "App". I mention it because we could change the proto package namespaces during the final code refactor which might make more sense at that point. @julienrbrt WDYT?

Reply:
I didn't realize https://github.com/ignite/cli/pull/3544 was targeting this branch.
I'll have a look at it now too 👍🏾